pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -186,15 +191,15 @@ unqualified-search = true`)
 	assert.Nil(t, err)
 	assert.Equal(t, 3, len(registries))
 
-	unqRegs := FindUnqualifiedSearchRegistries(registries)
+	var unqRegs []Registry
```

Comment:
(Non-blocking: Is the declaration necessary? I thought `unqregs, err := Find…` works if one of the variables is new.)

Reply:
You're absolutely right. Cleaned up with the next update to this PR. Thanks!